pkg/classification/interfaces/interfaces.go

```diff
@@ -92,6 +93,20 @@ func New(config Config) (*Classifier, error) {
 			}
 			preparedRecipe.URLS = append(preparedRecipe.URLS, preparedRecipeURL)
 		}
+
+		for _, ignoredRecipeURL := range recipe.ExcludeURLS {
```

Comment:
I think we should pick either ignored or excluded and not mix the two terms.
```suggestion
		for _, excludedRecipeURL := range recipe.ExcludeURLS {
```